doc/api_assets/style.css

```diff
@@ -330,6 +330,23 @@ hr {
   margin-top: .666em;
 }
 
+#toc .stability_0,
+#toc .stability_0 a,
+#toc .stability_0 a:link,
+#toc .stability_0 a:active {
+  color: #98001c;
+}
+
+#toc .stability_0 a {
+  text-decoration: line-through;
```

Comment:
@nodejs/collaborators Would we be ok with deprecations having a strike-through in the TOC or should it be something else?

This also make them red, but we do also need to consider not everyone can see colors.